### PR TITLE
[metadata manager m8s][query translator] calculate total size of obje…

### PR DESCRIPTION
**Pull request by josephjacobmorris**
**What type of PR is this?**
> /kind enhancement

**What this PR does / why we need it**:

This PR helps in calculating various aggregation data about the backend such as totalsize of bucket, number of objects and buckets.

**Test Report Added?**:
> /kind TESTED

**Test Report**:
 
image

### CODE REVIEW

**Review by josephjacobmorris — COMMENTED** [review]
Self-Review done

**Review by josephjacobmorris — COMMENTED** [review]
Self-Review done

File: metadata/pkg/model/model.go [hunk 1]
```diff
@@ -21,11 +21,12 @@ import (
...
-	Id          bson.ObjectId `json:"id,omitempty" bson:"_id,omitempty"`
-	BackendName string        `json:"backendName,omitempty" bson:"backendName,omitempty"`
-	Type        string        `json:"type,omitempty" bson:"type,omitempty"`
-	Region      string        `json:"region" bson:"region"`
-	Buckets     []MetaBucket  `json:"buckets,omitempty" bson:"buckets,omitempty"`
+	Id              bson.ObjectId `json:"id,omitempty" bson:"_id,omitempty"`
+	BackendName     string        `json:"backendName,omitempty" bson:"backendName,omitempty"`
+	Type            string        `json:"type,omitempty" bson:"type,omitempty"`
+	Region          string        `json:"region" bson:"region"`
+	Buckets         []MetaBucket  `json:"buckets,omitempty" bson:"buckets,omitempty"`
```
Comment (devanshjain7) [inline metadata/pkg/model/model.go:28]: please consider the latest model as there are changes in few field names and types

**Review by PravinRanjan10 — APPROVED** [review]
LGTM

**Pull request merged**